posthog/views.py

```diff
@@ -27,8 +27,17 @@
 )
 from posthog.version import VERSION
 
+# These are the same for now, but we'll keep them separate just in case we need to tweak
 ROBOTS_TXT_CONTENT = (
-    "User-agent: *\nDisallow: /shared_dashboard/" if settings.MULTI_TENANCY else "User-agent: *\nDisallow: /"
+    """
+    User-agent: *
+    Disallow: /
```

Comment:
Unsure this is what we want. This would prevent `app.posthog.com` from appearing on Google, which I'm not certain is what we want.

How about:

```suggestion
    Allow: /$
    Disallow: /
```

this should allow the root and nothing else. i.e. app.posthog.com but not app.posthog.com/insights etc

Reply:
Yes, I think this should be fine.

Google treats subdomains as their own entities, so there's no risk of this impact other parts of the site.

While we're at it, though, I think we should make similar changes to these subdomains:

- app-static.posthog.com
- t.posthog.com
- or.posthog.com
- playground.posthog.com
- discourse.posthog.com

They're less significant, but they're also generating a bit of crawling noise that's unnecessary.